lumicks/pylake/force_calibration/tests/test_active_calibration.py

```diff
@@ -139,3 +139,63 @@ def test_bias_correction():
 
     assert fit_biased.params["Bias correction"].value is False
     assert fit_debiased.params["Bias correction"].value is True
+
+
+def test_faxen_correction_active():
+    """Active calibration should barely be affected by surface corrections for the drag coefficient.
+    However, the interpretation of gamma_ex, which may be carried over to the other calibration
+    *is* important, so this should be covered by a specific test."""
```

Comment:
The test docstrings are excellent! Great to have the purpose of the test explained!